### DISCUSSION TITLE
ait-wiki/blocks/console/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#5] [General]
# ait-wiki/blocks/console/

The TARDIS console is the main control unit of the TARDIS, which can be interacted with by a player or another entity. The TARDIS should always be piloted by 6 fully-trained TARDIS pilots as to allow for a smooth journey.
Out-of-universe, the console is a collection of Control Entities which are located around the console block that do different functions.
How do I create a console? To create a console, craft a Console Generator.

https://pavatus.github.io/ait-wiki/blocks/console/

**Comment by Rasttny** [comment]
Increment control: 1-10-100-1000-10000

**Comment by dootdoot8686** [comment]
how do i rotate the console?

**Comment by Loqor** [comment]
you dont.

**Comment by Beridiusthewise** [comment]
how do you change the console

**Reply by Addi3** [reply]
https://amblelabs.github.io/ait-wiki/blocks/monitor/#tardis-settings-customisation and  https://amblelabs.github.io/ait-wiki/blocks/monitor/#tardis-settings-customisation

**Reply by Beridiusthewise** [reply]
thx so much

**Comment by ebusch8** [comment]
I can't place the coral growth on the soul sand to grow my console

**Reply by EchoDidAThing1048** [reply]
The area around to soul sand must be lit up

**Reply by ebusch8** [reply]
Like, lit up from the engine?

**Reply by ebusch8** [reply]
engine powers the tardis lights, does that count?

**Reply by EchoDidAThing1048** [reply]
**Torches**

**Reply by ebusch8** [reply]
got it

**Comment by Digli-exe** [comment]
Mine isn't lit/emitting light
